pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1078,6 +1113,8 @@ func (n *NetworkPolicyController) getAddressGroupMemberSet(g *antreatypes.Addres
 func (n *NetworkPolicyController) getClusterGroupMembers(group *antreatypes.Group) (controlplane.GroupMemberSet, []controlplane.IPBlock) {
 	if len(group.IPBlocks) > 0 {
 		return nil, group.IPBlocks
+	} else if group.Selector != nil && group.Selector.NodeSelector != nil {
+		return n.addNodeSelectorMemberSet(group.Selector.NodeSelector), nil
```

Comment:
this is for clusterGroup, but I didn't see you add nodeSelector to that struct

Reply:
Yeah, mistake, deleted.